challenge_9/python/sarcodian/src/challenge_9.py

```diff
@@ -0,0 +1,23 @@
+def square_n_sort(L):
+
+    L_square = []
+    L_sorted = []
+    count = len(L)
+
+    if L[0] >= 0:
+        for i in L:
+            L_square.append(i**2)
+        return L_square
+    
```

Comment:
I know it isn't, but trying to practice / try to be efficient. I figure the while loop is only needed with negative numbers. so if the numbers are all positive, then why bother comparing and appending, and then reversing. Imagine if the list had 500 positive numbers.